network/network.go

```diff
@@ -36,6 +40,19 @@ const MainNetNetwork Network = "mainnet"
 const SiriusNetwork Network = "sirius"
 const StandaloneNetwork Network = "standalone"
 
+func NewNetworkFromId(networkId uint64) (Network, error) {
+	switch networkId {
+		case 161:
+			return StandaloneNetwork, nil
+		case 162:
+			return StandaloneNetwork, nil
+		case 163:
+			return StandaloneNetwork, nil
```

Comment:
[critical] Aiiiiii Gabriiiii :) Should be Sirius and Mainnet.

Reply:
ups ! thanks for your sharp eye